src/Integration.UnitTests/MefServices/VsSessionHostTests.cs

```diff
@@ -339,16 +353,14 @@ public void VsSessionHost_ResetBinding_ErrorInReadingBinding()
         public void VsSessionHost_IServiceProvider_GetService()
         {
             // Arrange
-            var testSubject = new VsSessionHost(this.serviceProvider, this.sonarQubeServiceMock.Object,
```

Comment:
Tidy up - all of the other tests in this class had previously been changed to use the `CreateTestSubject` method, but not this one; not sure why.